php/mailSender.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+include 'debugging.php';
+require('/home/notfound/connect.php');
+require("functionsIDD.php");
+
+$from = trim($_POST['from']);
+$subject = trim($_POST['subject']);
+$compose = trim($_POST['compose']);
+$isValid = true;
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <title>Send Emails</title>
+</head>
+<body>
+<div class="container">
+    <div class="jumbotron banner">
+        <h1>Confirmation</h1>
+    </div>
+
+    <?php
+    //Makes sure all the fields are not empty
+        if(!validReqText($from)){
+            echo "<p>Error: The From field was left empty</p>";
+            $isValid = false;
+        }
+        else{
+            echo "<p>From: $from</p>";
+        }
+
+        if(!validReqText($subject)){
+            echo "<p>Error: The Subject field was left empty</p>";
+            $isValid = false;
+        }
+        else{
+            echo "<p>From: $subject</p>";
+        }
+
+        if(!validReqText($compose)){
+            echo "<p>Error: The Compose field was left empty</p>";
+            $isValid = false;
+        }
+        else{
+            echo "<p>From: $compose</p>";
+        }
+
+        //this will be in a loop
+        if($isValid){
+            $emailSend = $from;
+
+            $email_body = "I Day Dream --\r\n";
+            $email_body .= "$compose \r\n";
+
+
+
+            $email_subject = $subject;
+            $to = "olegrovin@gmail.com";//The active user goes here
```

Comment:
I tried this with my email but never got the email. Everything is uploaded on cpanel, not sure if it should be sending or not.